### PR TITLE
Added new recommendations structure. Isn't used yet.

### PR DESCRIPTION
**Pull request by p0psicles**
Will need some consecutive PR's to get this all to work.
Will change anidbhttp client libs next.

### CODE REVIEW

**Comment by SickRage2** [comment]
Thanks for the pull request! Before a real human comes by, please make sure your PR has all of the below criteria checked
- [x] Give a description on what the PR is for.
- [x] Make sure your PR is based on the DEVELOP branch
- [x] Don't send big changes all at once. Split up big PRs into multiple smaller PRs that are easier to manage and review

Please make sure you also read [contribution guide](https://github.com/SickRage/SickRage/blob/master/contributing.md) and followed all of the steps.

Thanks!

File: sickrage/show/recommendations/imdb.py [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+                show['image_url_large'] = self.change_size(image['src'], 3)
+                show['image_path'] = ek(posixpath.join, 'images', 'imdb_popular', ek(os.path.basename, show['image_url_large']))
+
+                self.cache_image(show['image_url_large'])
+
+            td = row.find("td", {"class": "title"})
+
+            if td:
+                show['name'] = td.find("a").contents[0]
+                show['imdb_url'] = "http://www.imdb.com" + td.find("a")["href"]
```
Comment (miigotu) [inline sickrage/show/recommendations/imdb.py:58]: `akas.imdb.com`

File: sickrage/show/recommendations/recommended.py [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+        self.votes = votes
+        self.title = title
+        self.image_href = image_href
+        self.image_src = image_src
+        self.indexer_id = indexer_id
+        self.indexer = indexer
+        self.show_in_list = None
+        
+        # Check if the show is currently already in the db
+        self.show_in_list = self.indexer_id in [ show.indexerid for show in sickbeard.showList if show.indexerid ]
```
Comment (miigotu) [inline sickrage/show/recommendations/recommended.py:35]: no space after `[` or before `]`

File: sickrage/show/recommendations/imdb.py [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+                else:
+                    show['outline'] = u''
+
+                popular_shows.append(show)
+
+        return popular_shows
+
+    @staticmethod
+    def change_size(image_url, factor=3):
+        match = re.search("^(.*)V1._(.{2})(.*?)_(.{2})(.*?),(.*?),(.*?),(.*?)_.jpg$", image_url)
```
Comment (miigotu) [inline sickrage/show/recommendations/imdb.py:92]: This should be a raw string, `re.search(r"`

File: sickrage/show/recommendations/__init__.py [hunk 4]
```diff
@@ -0,0 +1,3 @@
+# coding=utf-8
+
+__all__ = ['anidb','imdb','trakt']
```
Comment (miigotu) [inline sickrage/show/recommendations/__init__.py:3]: always a space after a comma

**Comment by miigotu** [comment]
Can worry about those small things later.

File: sickrage/show/recommendations/imdb.py (hunk 1 above)
Comment (p0psicles) [inline sickrage/show/recommendations/imdb.py:58]: @miigotu www also works, if you have a good reason to change it with akas, i'll add it in a PR to include these comments.

Comment (OmgImAlexis) [inline sickrage/show/recommendations/imdb.py:58]: @p0psicles http://forum.kodi.tv/showthread.php?tid=4153

Comment (miigotu) [inline sickrage/show/recommendations/imdb.py:58]: Not only that, we have learned through experience that akas has less hotlinking/scrape protection than www.

**Comment by p0psicles** [comment]
https://github.com/SickRage/SickRage/pull/893

File: sickrage/show/recommendations/__init__.py (hunk 4 above)
Comment (labrys) [inline sickrage/show/recommendations/__init__.py:3]: This shouldn't even be in `__init__.py` it should be an empty file.

**Pull request merged**